feature/visited/src/main/kotlin/uk/govuk/app/visited/data/VisitedItemsTransformer.kt

```diff
@@ -0,0 +1,78 @@
+package uk.govuk.app.visited.data
+
+import uk.govuk.app.visited.data.model.VisitedItem
+import uk.govuk.app.visited.ui.model.VisitedUi
+import java.time.LocalDate
+
+class VisitedItemsTransformer(
```

Comment:
This could probably be a stand alone (or extension) function rather than a class.

Reply:
Fixed up

Reply:
I'm still seeing it as a class

Reply:
That's odd 🤔 You don't see `fun transformVisitedItems(visitedItems: List<VisitedItemUi>, todaysDate: LocalDate): Map<String, List<VisitedUi>> {` [here](https://github.com/alphagov/govuk-mobile-android-app/pull/91/commits/8a221f34054365ebf858b17937c2e26f1c3eab51#diff-40c77ccd20d05237a073377c0cf1c4f5eb18ed2b5fa79bea8dc14397f3c45295)